.circleci/config.yml

```diff
@@ -35,20 +35,25 @@ jobs:
       - run:
           name: Upgrade pip, setuptools, and wheel before installing other dependencies
           command: python -m pip install --upgrade pip setuptools wheel
-      
+
       - run:
           name: 'Install required packages'
           command: python scripts/install_requirements.py --pinned False
 
       - run:
           name: "Build Documentation"
-          command: python scripts/build_documentation.py --documentation-path=docs --output-path=docs/_build/html
+          command: |
+            if [[ "${CIRCLE_BRANCH}" =~ ^main$|^[0-9]+\.[0-9]$|^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              bash python scripts/build_documentation.py --documentation-path=docs --output-path=docs/_build/html
+            else
+              make doc
```

Comment:
What I understand here is: if the branch is main or a release version then build the versioned docs, otherwise just the single-version-webpage. But then how would we know if we break the versioned docs on a PR?

Reply:
That's fair. In combination with the other PR that completely addresses my concern.